index.js

```diff
@@ -136,6 +136,27 @@ exports.array = function(val) {
 };
 
 /**
+ * Check if an array contains only items from another array
+ * @param  {Array} arrayOfAcceptedValues
+ * @return {Function}
+ */
+exports.arrayContaining = function(arrayOfAcceptedValues) {
+    return function(val) {
+      var bool = true;
+      if(val && val instanceof Array) {
+        val.forEach(value => {
```

Comment:
It'd be nice to use `.find()` here to make this a one liner and exit the loop early when we find a value which isn't contained in the other array... but support isn't great and we're not polyfilling much on the frontend for our newer solutions so lets leave this as-is :)